CONTRIBUTING.md

```diff
@@ -1,181 +1,163 @@
-# Quire Contributor Guidelines
+# Quire Documentation Contributor Guidelines
 
-**Thank you for your interest in contributing to Quire!**
+>**All are welcome.** As an open-source community, Quire is committed to providing a safe, welcoming, transparent, and inclusive environment for all our community members and those wishing to become involved. Please see our **[Code of Conduct](https://github.com/thegetty/quire/blob/main/CODE_OF_CONDUCT.md)** for more on the expectations and protections for our community members.
 
-Developed by Getty, Quire is a new, multiformat publishing tool available for immediate access and use. Quire is in a limited beta, © J. Paul Getty Trust, and not yet released as open-source software. **For a free license to use Quire, please complete [this form](http://bit.ly/quire-signup).**
-
-Striving toward becoming a fully open-source project by spring 2022, we encourage contributions from our community members. No matter what level of experience you have, we welcome all contributions, big and small.
-
-There are [many ways to contribute](#identify-a-contribution-to-make) including fixing bugs, improving the documentation, and creating a new feature or theme. Please read the following guidelines carefully. Our goal is to make the process as effective and transparent as possible, and to ensure that your every contribution can become part of Quire. Thank you for taking the time.
-
-🗣**CALLING ALL FIRST-TIMERS!** — We can’t stress enough that Quire is open to contributors at all levels. That could mean making a modest fix, like correcting a typo or fixing a broken link, or more intensive contributions like adding Quire CSS styles and JavaScript, or even copyediting one of our documentation pages. Here are a few resources especially for you:
+**Thank you for your interest in contributing to Quire!** Here is the process we would like you to follow.
```

Comment:
> Here is the process we would like you to follow

Could we soften this a little? Just felt a little too formal and prescriptive to me. Also maybe make the link list below a numbered list so that it reads more like a "process".